kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/RecordEncryptionFilter.java

```diff
@@ -57,6 +61,27 @@ public class RecordEncryptionFilter<K>
         this.encryptionManager = encryptionManager;
         this.decryptionManager = decryptionManager;
         this.filterThreadExecutor = filterThreadExecutor;
+        checkCipherSuite();
```

Comment:
realised we are in the Filter, I think this needs to move up into the Factory init to fail early